lib/tilex/notifications/notifications.ex

```diff
@@ -1,4 +1,6 @@
 defmodule Tilex.Notifications do
+  @timezone Application.get_env(:tilex, :date_display_tz, "America/Chicago")
```

Comment:
@tmock12 I think that we should avoid to use module attributes for `Application.get_env`... or anything else that evaluates on compilation time. Chiefly when their config value is a `System.get_env`, like:

```elixir
config :tilex, :date_display_tz, System.get_env("DATE_DISPLAY_TZ")
```

Elixir won't compile again this file if the only change is a new `DATE_DISPLAY_TZ` value, so it could cause a lot of confusion OMHO.

Reply:
👍 you're right. I didn't notice `:date_display_tz` was being set via `System.get_env/3`